Apps/APN-UIKit/APN UIKit/View/DashboardViewController.swift

```diff
@@ -172,3 +177,14 @@ class DashboardViewController: BaseViewController {
         UNUserNotificationCenter.current().add(request, withCompletionHandler: nil)
     }
 }
+
+extension DashboardViewController: InlineMessageDelegate {
+    func onInlineCustomButtonAction(message: Message, action: String, name: String) {
```

Comment:
The [tech proposal's public API](https://www.notion.so/custio/Mobile-inline-in-app-messages-technical-proposal-7efb22e2af1c40449c72f9cf73770584?pvs=4#d722b4ab9b854d84b86d744cd840d307) specifies the function signature to be `onActionClick(message: InAppMessage, actionValue: String, actionName: String)`. Could you rename the signature to match this?